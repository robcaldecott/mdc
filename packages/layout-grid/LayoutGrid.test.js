import React from "react";
import { LayoutGrid, LayoutGridCell } from ".";
import renderer from "react-test-renderer";

it("renders", () => {
  expect(
    renderer.create(
      <LayoutGrid>
        <LayoutGridCell
          span={1}
          desktop={2}
          tablet={3}
          mobile={4}
          order={2}
          align="middle"
        >
          Cell 1
        </LayoutGridCell>
        <LayoutGridCell>
          <LayoutGrid nested>
            <LayoutGridCell>1</LayoutGridCell>
            <LayoutGridCell>2</LayoutGridCell>
            <LayoutGridCell>3</LayoutGridCell>
          </LayoutGrid>
        </LayoutGridCell>
      </LayoutGrid>
    )
  ).toMatchSnapshot();
});
