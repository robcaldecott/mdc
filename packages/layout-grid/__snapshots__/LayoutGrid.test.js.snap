// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<div
  className="mdc-layout-grid"
>
  <div
    className="mdc-layout-grid__inner"
  >
    <div
      className="mdc-layout-grid__cell mdc-layout-grid__cell--span-1 mdc-layout-grid__cell--span-2-desktop mdc-layout-grid__cell--span-3-tablet mdc-layout-grid__cell--span-4-mobile mdc-layout-grid__cell--order-2 mdc-layout-grid__cell--align-middle"
    >
      Cell 1
    </div>
    <div
      className="mdc-layout-grid__cell"
    >
      <div
        className="mdc-layout-grid__inner"
      >
        <div
          className="mdc-layout-grid__cell"
        >
          1
        </div>
        <div
          className="mdc-layout-grid__cell"
        >
          2
        </div>
        <div
          className="mdc-layout-grid__cell"
        >
          3
        </div>
      </div>
    </div>
  </div>
</div>
`;
